tests/run-make/const_fn_mir/rmake.rs

```diff
@@ -0,0 +1,8 @@
+// The `needs-unwind -Cpanic=abort` gives a different MIR output.
```

Comment:
This should have been `//@ needs-unwind`. `# needs-unwind` in Makefile tests ignores the test when `-Cpanic=abort` is the default, like with cg_clif or when compiling for wasm.

Reply:
Worked around this in https://github.com/rust-lang/rustc_codegen_cranelift/commit/e24117653571901c8e1a2212fb93dd0cc4401aaf

Reply:
Sorry, fixing it and reverting your mir changes as well.